prombench/Makefile

```diff
@@ -7,37 +7,37 @@ deploy: nodepool_create resource_apply
 clean: resource_delete nodepool_delete
 
```

Comment:
We should use the `PROVIDER` makefile variable for both funchbench and prombench Makefiles. ( Currently can be need in #384 Makefile for prombench)